README.md

```diff
@@ -107,54 +110,79 @@ keys_gen.sh can be used to generate random passwords for each service.env.
 
 ### Generating random passwords using keys_gen.sh
 
-Running keys_gen.sh will generate random passwords for the all http servers.
+1. Generating demo certificate authority KeyPair and certificate
 
-```
-$ cd <fdo-pri-src>/component-samples/demo/scripts
-$ ./keys_gen.sh
-```
+    ```shell
+      bash demo-ca.sh
+    ```
 
-A message "Key generation completed." will be displayed on the console.
+    **NOTE**: Configure the properties of `demo-CA` by updating `root-ca.conf`.
 
-Credentials for each service will be generated the creds directory `<fdo-pri-src>/component-sample/demo/`.
+2. Generating Server and Client Keypair and certificates.
 
-```
-creds/aio/service.env
-creds/manufacturer/service.env
-creds/owner/service.env
-creds/reseller/service.env
-creds/rv/service.env
-```
+    ```shell
+    bash web_csr_req.sh
+    bash user_csr_req.sh
+    ```
 
-Replace the service.env in the demo folders with the generated ones in the creds folder.
+    **NOTE**: Both Server and Client certificate are signed by the previous generated demo-CA. Moreover, we can configure the properties of Server and Client certificate by updating `web-server.conf` and `client.conf` respectively.
+
+
+3. Running keys_gen.sh will generate random passwords for the all http servers and creates `secrets` folder containing all the required `.pem` files of Client, CA and Server component.
+
+    ```
+    $ cd <fdo-pri-src>/component-samples/demo/scripts
+    $ ./keys_gen.sh
+    ```
+
+    A message "Key generation completed." will be displayed on the console.
+
+    Credentials will be stored in the `secrets` directory within `<fdo-pri-src>/component-sample/demo/scripts`.
+
+4. Copy both `secrets/` and `service.env` file from  `<fdo-pri-src>/component-sample/demo/scripts`  folder to the individual components.
+
+    ```
+    $ cd <fdo-pri-src>/component-samples/demo/scripts
+    $ cp -r ./secrets/. ../replace-with-FDO-component
+    $ cp service.env ../replace-with-FDO-component
```

Comment:
do we need to clarify replace-with-fdo-component

Reply:
Done.